python/dgl/graphbolt/minibatch_sampler.py

```diff
@@ -108,11 +100,72 @@ class MinibatchSampler(IterDataPipe):
     >>> data_pipe = data_pipe.map(add_one)
     >>> list(data_pipe)
     [tensor([1, 2, 3, 4]), tensor([5, 6, 7, 8]), tensor([ 9, 10])]
+
+    7. Heterogeneous node/edge IDs.
+    >>> ids = {
+    ...     ("user", "like", "item"): gb.ItemSet(torch.arange(0, 5)),
+    ...     ("user", "follow", "user"): gb.ItemSet(torch.arange(5, 10)),
```

Comment:
When we ask users to provide dataset for heterogeneous training.

For different node types, do we ask users to provide ids always from 0, or globally continuous?

Reply:
I don't think we have any assumption or limitation on node ids. ids always could be arbitrary.

Reply:
The itemset should not have any assumption, but as example, it should show what's we recommend to users.